models/public/hrnet-w32-human-pose-estimation/hrnet-w32-human-pose-estimation.md

```diff
@@ -0,0 +1,85 @@
+# hrnet-w32-human-pose-estimation
+
+## Use Case and High-Level Description
+
+The `HRNet-W32` model is one of the [HigherHRNet](https://arxiv.org/pdf/1908.10357).
+`HigherHRNet` is a novel bottom-up human pose
+estimation method for learning scale-aware representations using high-resolution feature pyramids. The network uses HRNet as backbone, followed by one or more deconvolution modules to generate multi-resolution and high-resolution heatmaps. For every person in an image, the network detects a human pose: a body skeleton consisting of keypoints and connections between them. The pose may contain up to 17 keypoints: ears, eyes, nose, shoulders, elbows, wrists, hips, knees, and ankles.
+This is PyTorch\* implementation pretrained on COCO dataset.
+For details about implementation of model, check out the [HigherHRNet: Scale-Aware Representation Learning for Bottom-Up Human Pose Estimation](https://github.com/HRNet/HigherHRNet-Human-Pose-Estimation) repository.
+
+## Specification
+
+| Metric            | Value                  |
+|-------------------|------------------------|
+| Type              | Human pose estimation  |
+| GFLOPs            | 92.8364                |
+| MParams           | 28.6180                |
+| Source framework  | PyTorch\*              |
+
+## Accuracy
+
+| Metric | Original model | Converted model |
+| ------ | -------------- | --------------- |
+| Average Precision (AP)  | 65,02%          | 65,02%           |
```

Comment:
```suggestion
| Metric                            | Original model | Converted model |
| ------ -------------------| ----------------- | ------------------- |
| Average Precision (AP)  | 65,02%             | 65,02%                 |
```
P.S. I know that it  does not affect markdown representation, but it is more human-readable in editor mode